access_application.go

```diff
@@ -110,13 +110,42 @@ const (
 	AccessApplicationScimAuthenticationSchemeHttpBasic        AccessApplicationScimAuthenticationScheme = "httpbasic"
 	AccessApplicationScimAuthenticationSchemeOauthBearerToken AccessApplicationScimAuthenticationScheme = "oauthbearertoken"
 	AccessApplicationScimAuthenticationSchemeOauth2           AccessApplicationScimAuthenticationScheme = "oauth2"
+	ScimAuthenticationAccessServiceToken                      AccessApplicationScimAuthenticationScheme = "access_service_token"
```

Comment:
```suggestion
	AccessApplicationScimAuthenticationAccessServiceToken                      AccessApplicationScimAuthenticationScheme = "access_service_token"
```

Reply:
you'll want to update the rest of the usage of this as well to make sure it all still works.